packages/account-v2/src/modules/src/AddressFields/validations.ts

```diff
@@ -53,15 +35,6 @@ export const addressDetailValidations = (countryCode: string, isSvg: boolean) =>
         }),
     addressPostcode: Yup.string()
         .max(20, 'Please enter a postal/ZIP code under 20 characters.')
-        .matches(regexChecks.addressPostcode, 'Only letters, numbers, space and hyphen are allowed.')
-        .when({
-            is: () => countryCode === 'gb',
```

Comment:
Removing this business logic since its not required anymore. Discussion here:  https://deriv-group.slack.com/archives/C0647GW0QRW/p1712036803676689